nautobot/extras/management/__init__.py

```diff
@@ -332,13 +163,14 @@ def export_statuses_from_choiceset(choiceset, color_map=None, description_map=No
     return choices
 
 
-def create_custom_statuses(
+def create_custom_metadata_instances(
     app_config=None,  # unused
     verbosity=2,
     interactive=True,
     using=DEFAULT_DB_ALIAS,  # unused
     apps=global_apps,
     models=None,
+    metadata_model=None,
```

Comment:
Having a `models` and a `metadata_model` is confusing. Please update the docstring to make it more clear what this is doing and perhaps make it a private function. Then we can use wrapper functions for `populate_status_choices` and `populate_role_choices`? `create_custom_metadata_instances` doesn't really indicate `Role` and `Status` to me.